app/Http/Controllers/User/BookController.php

```diff
@@ -19,19 +21,26 @@ class BookController extends Controller
 
     protected $media;
 
+    protected $review;
+
     protected $with = [
         'medias',
         'categories',
         'owners',
         'reviews',
     ];
 
-    public function __construct(BookRepository $book, CategoryRepository $category, BookCategoryRepository $bookCategory, MediaRepository $media)
-    {
+    public function __construct(BookRepository $book,
+    CategoryRepository $category,
+    BookCategoryRepository $bookCategory,
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.FirstParamSpacing</br>**Severity**: error</br>**File**: app/Http/Controllers/User/BookController.php</br>The first parameter of a multi-line function declaration must be on the line after the opening bracket